extensions/amp-carousel/0.2/amp-carousel.js

```diff
@@ -106,6 +106,13 @@ class AmpCarousel extends AMP.BaseElement {
 
     /** @private {?ChildLayoutManager} */
     this.childLayoutManager_ = null;
+
+    /**
+     * Whether or not to skip a slideChange when the index changes. Used to
+     * avoid triggering a slideChange on the initial render.
+     * @private {boolean}
+     */
+    this.skipNextSlideChange_ = true;
```

Comment:
It's not possible to use the check that `currentIndex_` is null instead of adding an instance variable here?

Reply:
Good call, made the change and also fixed #24701 in the process.